buildroot-external/board/generic-aarch64/cmdline.txt

```diff
@@ -0,0 +1 @@
+set CMDLINE="rootwait zram.enabled=1 zram.num_devices=3 apparmor=1 security=apparmor systemd.machine_id=$MACHINE_ID cgroup_enable=memory fsck.repair=yes"
```

Comment:
This should only contain boot arguments which the user should adjust. For now I would leave it empty. Also, I think set is not strictly necessary correct? Can this be:

```suggestion
cmdline=""
```